### ISSUE TITLE
Quest Bug

### ISSUE THREAD

**Issue by maxminoS** [Wynncraft/Issues#1648]
Wynn Excavation Site A Quest

I have just completed the following quest, though, I cannot complete it due to the criteria required. It is said that you need to submit the ancient encryption to the scientist, and I did that. My friend claims that he got it back, but I did not. And when I went to the tomb and killed the Pharaoh, I went back with him to Excavator Vade (the character). But then it asked me again, to give back the encryption he gave me. Here are some screenshots to prove the events:

![screen shot 2015-02-21 at 1 16 44 pm](https://cloud.githubusercontent.com/assets/11131914/6313132/1aeea2ee-b9cf-11e4-9a73-b0428095b55c.png)
![screen shot 2015-02-21 at 1 16 39 pm copy](https://cloud.githubusercontent.com/assets/11131914/6313134/1b5b3d46-b9cf-11e4-9bf3-5991c1e584a9.png)
![screen shot 2015-02-21 at 1 16 39 pm](https://cloud.githubusercontent.com/assets/11131914/6313133/1b55a390-b9cf-11e4-81f2-437125eb47e3.png)

**Comment by Trixter313** [comment]
You can try typing the command /fixquests but it's really unfinished and most likely won't work.

**Comment by maxminoS** [comment]
I just tried it, it didn't work. "No missing quest item"

**Comment by DesiredLover39** [comment]
The same thing is happening with me. I went to Vade and got the enscription. I took it to Tessa and she gave it back to me. I went back to Vade and gave it to him. It let me carry on and I went on but now it is saying the same thing.
![minecraft wynnexcavation site a](https://user-images.githubusercontent.com/33137406/32079718-bc272200-ba60-11e7-95b6-52e61cbb9f93.png)
![wynncraft bank chest](https://user-images.githubusercontent.com/33137406/32079723-c54b1f08-ba60-11e7-8020-f74a4944738d.png)
![fixquests command](https://user-images.githubusercontent.com/33137406/32079734-cc842efe-ba60-11e7-86fb-39419b79dfd9.png)
